src/librbd/ImageWatcher.cc

```diff
@@ -1382,6 +1384,8 @@ void ImageWatcher<I>::process_payload(uint64_t notify_id, uint64_t handle,
   auto ctx = new Watcher::C_NotifyAck(this, notify_id, handle);
   bool complete;
 
+  ldout(cct, 10) << this << " process_payload "  << payload->get_notify_op() << dendl;
```

Comment:
I am not sure how useful it is. We have already a message in `handle_payload`, which is called just below and usually provides much more information. In what case do you think it is useful?

Reply:
I used this to identify what payload { AsyncComplete or UpdateFeatures was being processed and to judge by it has the requested later or not, maybe we can add just ` payload->get_notify_op() ` to handle_payload info itself